src/lib/y2users/password.rb

```diff
@@ -65,7 +74,7 @@ def initialize(config, name, value: nil, last_change: nil, minimum_age: nil,
       account_expiration: nil, source: :unknown)
```

Comment:
I know we already have tons of keyword arguments here, but having `value` and not `plain_value` looks wrong. In almost all cases you will initialize the object with one of those two arguments (all the others are less common).

Reply:
Moreover, it would maybe be useful to have some method like `#any_value?` to check if either of both is set.